src/style.css

```diff
@@ -0,0 +1,76 @@
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.input {
+  width: 100%;
+  margin-bottom: 20px;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: inherit;
+}
+
+.form:first-child {
+  margin-top: 20px;
+}
+
+.form:last-child {
+  margin-bottom: 0;
+}
+
+.search {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  background-position: left 25px top 24px;
+  padding-left: 62px;
+  background-size: 19px 19px;
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.small_search {
+  background-position: left 12px top 14px;
+  padding-left: 33px;
+  background-size: 11px 11px;
+  height: 42px;
+  font-size: 14px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: "Book";
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: "Heavy";
+  font-weight: 800;
+}
+
+.big_search:hover {
+  font-style: "Book";
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
```

Comment:
`big_search` class looks redundant, you can move this styles under general class `search`